### ISSUE TITLE
Helm 3.3.0 reporting it was built from a "dirty" git tree state

### ISSUE THREAD

**Issue by bacongobbler** [helm/helm#8620]
This is likely related and was likely fixed with https://github.com/helm/helm/pull/8544, but I'm reporting it to verify if this has been fixed for Helm 3.3.1/3.4.0.

```
><> gofish list helm
NAME    VERSION LINKED
helm    3.2.4   false
helm    3.3.0   true
><> helm version
version.BuildInfo{Version:"v3.3.0", GitCommit:"8a4aeec08d67a7b84472007529e8097ec3742105", GitTreeState:"dirty", GoVersion:"go1.14.7"}
```

**Comment by mattfarina** [comment]
That would explain why I couldn't replicate it on master. Good catch.

**Comment by bacongobbler** [comment]
fixed in Helm 3.3.1
